views/default/elgg/ajax.js

```diff
@@ -0,0 +1,199 @@
+define(function (require) {
+	var $ = require('jquery');
+	var elgg = require('elgg');
+	var spinner = require('elgg/spinner');
+
+	var spinner_enabled = true;
```

Comment:
Every user of the API would still need to set it, just in case another user set it differently. The solution is to put the methods on an instance created locally.

```js
var ajax = new require('elgg/Ajax');
ajax.useSpinner = false;
...
```